new/LIT7427VitaSua.xml

```diff
@@ -0,0 +1,84 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7427VitaSua" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="la" xml:id="t1">Narratio de vita sua</title>
+                <title xml:lang="grc" xml:id="t2greek">CPG 6633</title>
+                <author ref="PRS7973PseudoD"/>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p>
+                    <listBibl type="clavis">
+                        <bibl type="CPG">
+                            <ptr target="bm:CPG2"/>
+                            <citedRange unit="item">6633</citedRange>
+                        </bibl>
+                    </listBibl>
+                </p>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Homily"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="la">Latin</language>
+                <language ident="grc">Greek</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-05-13">Created entity</change>
+            <change when="2025-05-18" who="CH">Corrections after reviews by Dorothea Reule and Nafisa Valieva</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
```

Comment:
I think it would be good to include the Geez incipit in this file (and maybe also the others, if possible, for exmaple from bm:Muthreich2022Dionysius, p. 32-33). Especially in combination with the latin title, I think this file might be currently a bit difficult to parse for anyone not familiar with the literature